Library/Homebrew/cmd/test-bot.rb

```diff
@@ -280,7 +280,7 @@ def single_commit?(start_revision, end_revision)
       def diff_formulae(start_revision, end_revision, path, filter)
         git(
           "diff-tree", "-r", "--name-only", "--diff-filter=#{filter}",
-          start_revision, end_revision, "--", path
+          "#{start_revision}..#{end_revision}", "--", path
```

Comment:
I don't know why but experiment shows this notation only working for `git log` instead of `git diff-tree`.